tests/nada-tests/src/array_attributes.py

```diff
@@ -14,6 +14,8 @@ def nada_main():
     b = na.NadaArray(np.array([]))
     c = na.NadaArray(np.array([na.rational(1.5)]))
 
+    a += Integer(0)
```

Comment:
Not a big fan of this :p
Same goes for all other instances of this.

Reply:
In this new version, programs without compute stop compiling. All these programs where there's this instance of a `+= Integer(0)` are programs that without this compute line would not compile. I know it's a tricky thing, but I don't know a better/faster way to handle it.